x/faucet/abci.go

```diff
@@ -0,0 +1,21 @@
+package faucet
+
+import (
+	"github.com/Sifchain/sifnode/x/faucet/keeper"
+	"github.com/Sifchain/sifnode/x/faucet/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// BeginBlocker check for infraction evidence or downtime of validators
+// on every begin block
+func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
+	if ctx.BlockHeight()%types.FaucetResetBlocks == 0 {
```

Comment:
We might need to revist this, initially while writting this part I had not considered the fact that ,if we a high trasaction numbers , the block time might not be a constant of 5 secs .

Although it might not be that important for a testnet

Reply:
I think the way it's written now is reliable for testnet. Do you think this is critical enough to change the module for testnet?